modules/KalturaSupport/components/logo.js

```diff
@@ -24,22 +24,22 @@
 							src: this.getConfig('img')
 						}).removeAttr("width height");
 				}
-				this.$el = $('<div />')
+				this.$el = $('<button />')
```

Comment:
why change it to button? did you check the change on old IEs and mobiles?

Reply:
@OrenMe Checked on All browsers and mobile devices and it works as expected, I changed it to a button because of this:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_button_role